daemons/dss-index/app.py

```diff
@@ -22,7 +23,10 @@ def dispatch_s3_indexer_event(event, context) -> None:
     if event.get("Event") == "s3:TestEvent":
         app.log.info("DSS index daemon received S3 test event")
     else:
-        process_new_indexable_object(event, logger=app.log)
+        bucket_name = event['Records'][0]["s3"]["bucket"]["name"]
+        key = urllib.parse.unquote(event['Records'][0]["s3"]["object"]["key"])
+
+        process_new_indexable_object(bucket_name, key, "aws", logger=app.log)
```

Comment:
you can use process_new_s3_indexable_object( event, logger) from event/handlers/index.py rather than process_new_indexable_object. Otherwise we should remove process_new_s3_indexable_obect and keep the event parsing logic in dss-index/app.py

Reply:
I can make the change if you'd like.